### PR TITLE
[Backport][ipa-4-9] ipa-idrange-fix

### PR DESCRIPTION
**Pull request by KoRORland**
backport of ipa-idrange-fix to ipa-4-9

### CODE REVIEW

**Comment by flo-renaud** [comment]
Hi @KoRORland 
Thanks for the backport. Can you add a temp commit to trigger the new test?

Note that the lint failure is unrelated to your backport and can be ignored:
```
************* Module ipalib.base
ipalib/base.py:472: [I0021(useless-suppression), ] Useless suppression of 'invalid-sequence-index')
************* Module ipatests.test_ipaplatform.test_tasks
ipatests/test_ipaplatform/test_tasks.py:17: [E1101(no-member), test_ipa_version] Instance of 'Version' has no 'version' member; maybe '_version'?)
ipatests/test_ipaplatform/test_tasks.py:24: [E1101(no-member), test_ipa_version] Instance of 'Version' has no 'version' member; maybe '_version'?)
ipatests/test_ipaplatform/test_tasks.py:34: [C0113(unneeded-not), test_ipa_version] Consider changing "not v3 == v4" to "v3 != v4")
```

**Comment by KoRORland** [comment]
added test commit, can't do the re-run

**Comment by KoRORland** [comment]
temp commit removed, selinux enforcing removed for latest and previous

File: ipatests/prci_definitions/nightly_ipa-4-9_previous.yaml [hunk 1]
```diff
@@ -1766,6 +1766,18 @@ jobs:
         timeout: 10800
         topology: *master_1repl
 
+  fedora-latest-ipa-4-9/test_idrange_fix:
```
Comment (flo-renaud) [inline ipatests/prci_definitions/nightly_ipa-4-9_previous.yaml:1769]: This line should read:  `fedora-previous-ipa-4-9/test_idrange_fix`

File: ipatests/prci_definitions/nightly_ipa-4-9_previous.yaml [hunk 2]
```diff
@@ -1766,6 +1766,18 @@ jobs:
...
+    requires: [fedora-latest/build]
```
Comment (flo-renaud) [inline ipatests/prci_definitions/nightly_ipa-4-9_previous.yaml:1770]: `requires: [fedora-previous-ipa-4-9/build]`

File: ipatests/prci_definitions/nightly_ipa-4-9_previous.yaml [hunk 3]
```diff
@@ -1766,6 +1766,18 @@ jobs:
...
+    priority: 50
+    job:
+      class: RunPytest
+      args:
+        build_url: '{fedora-latest-ipa-4-9/build_url}'
```
Comment (flo-renaud) [inline ipatests/prci_definitions/nightly_ipa-4-9_previous.yaml:1775]: ` build_url: '{fedora-previous-ipa-4-9/build_url}'`

File: ipatests/prci_definitions/nightly_ipa-4-9_previous.yaml [hunk 4]
```diff
@@ -1766,6 +1766,18 @@ jobs:
...
+        test_suite: test_integration/test_ipa_idrange_fix.py
+        template: *ci-ipa-4-9-latest
```
Comment (flo-renaud) [inline ipatests/prci_definitions/nightly_ipa-4-9_previous.yaml:1777]: `template: *ci-ipa-4-9-previous`

**Review by flo-renaud — COMMENTED** [review]
Hi @KoRORland 
please find inline comments

File: ipatests/prci_definitions/nightly_ipa-4-9_previous.yaml (hunk 4 above)
Comment (KoRORland) [reply ipatests/prci_definitions/nightly_ipa-4-9_previous.yaml]: fixed

File: ipatests/prci_definitions/nightly_ipa-4-9_previous.yaml (hunk 3 above)
Comment (KoRORland) [reply ipatests/prci_definitions/nightly_ipa-4-9_previous.yaml]: fixed

File: ipatests/prci_definitions/nightly_ipa-4-9_previous.yaml (hunk 2 above)
Comment (KoRORland) [reply ipatests/prci_definitions/nightly_ipa-4-9_previous.yaml]: fixed

File: ipatests/prci_definitions/nightly_ipa-4-9_previous.yaml (hunk 1 above)
Comment (KoRORland) [reply ipatests/prci_definitions/nightly_ipa-4-9_previous.yaml]: fixed

**Comment by flo-renaud** [comment]
ipa-4-9:

* 249b3ea33af974dd765d9e4e9046bf1dc0c0fb55 Add ipa-idrange-fix
* 624f5d60b47f7719ae02a4a632e2d87b347d94fc Add PR-CI definitions

**Pull request closed without merging**